code/game/objects/items/stacks/medical.dm

```diff
@@ -81,8 +76,18 @@
 			M.updatehealth()
 		else
 			user << "<span class='notice'>Medicine won't work on a robotic limb!</span>"
+			return
 	else
+		var/t_himself = "itself"
+		if(user.gender == MALE)
+			t_himself = "himself"
+		else if(user.gender == FEMALE)
+			t_himself = "herself"

```

Comment:
does \his not work in this situation or is this just silliness from someone who didn't know it existed